wrench/src/main.rs

```diff
@@ -442,211 +425,133 @@ fn main() {
 
     let mut show_help = false;
     let mut do_loop = false;
-
-    let queue_frames = thing.queue_frames();
-    for _ in 0 .. queue_frames {
-        let (width, height) = window.get_inner_size_pixels();
-        let dim = DeviceUintSize::new(width, height);
-        wrench.update(dim);
-
-        let frame_num = thing.do_frame(&mut wrench);
-        unsafe {
-            CURRENT_FRAME_NUMBER = frame_num;
-        }
-
-        wrench.render();
-        window.swap_buffers();
-    }
```

Comment:
Can you explain why this was a loop, and now it's not?

Reply:
I *think* that when being used for timing / profiling purposes, the idea here was to queue a set of initial frames to avoid initial timing spikes.